pycparser/c_parser.py

```diff
@@ -574,8 +574,11 @@ def p_pp_directive(self, p):
     def p_pppragma_directive(self, p):
         """ pppragma_directive      : PPPRAGMA
                                     | PPPRAGMA PPPRAGMASTR
+                                    | _PRAGMA LPAREN unified_string_literal RPAREN
```

Comment:
Can you add a comment explaining why the contents of `_PRAGMA` differ from `PPPRAGMA`?

Reply:
Well, it has a different syntax? 
![image](https://user-images.githubusercontent.com/56157703/211168138-e369bc49-8ee7-4960-a1fa-a2d70a856dad.png)

`#pragma` does not require parenthesis and strings as far as I know. Not sure what to explain?

Reply:
The comment is good, thanks! Please move it out of the docstring though - this doesn't work with PLY's yacc (see test error in the GitHub action)